package com.hpe.day8;

public class ExceptionEx01 {
	public static void main(String[] args) {
		try {
			int x = Integer.parseInt(args[0]); 
			int y = Integer.parseInt(args[1]); 
			
			int z = x/y; 
			
			System.out.println("Result is " + z);

		}catch(ArithmeticExceptione) {
			System.out.println(e);
		}
		
		System.out.println("This is some other business logic ");
	}
}
